src/events/AddReactionEventHandler.ts

```diff
@@ -1,8 +1,9 @@
-import { User, MessageReaction } from 'discord.js';
+import {User, MessageReaction, TextChannel} from 'discord.js';
```

Comment:
The code style requires spaces here.
You can install the vscode extention `ESLint` to get those marked automatically.
Also you can run `npm run lint` to fix those things.

Reply:
Weird, I ran npm run lint before committing

Also I use intellij idea which marks them as errors supposedely

Reply:
Yeah, lint doesn't catch missing spaces around curly braces. it only works with parentheses. There's probably some rule for that somewhere that we could enable, though.